com.zerobrane.studio.json

```diff
@@ -0,0 +1,54 @@
+/* Automatically generated from com.github.MrSyabro.zbstudio.yml, do not modify */
+{
+  "app-id": "com.zerobrane.studio",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "zbstudio",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "zbstudio",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pwd",
+        "sh ZeroBraneStudioEduPack-1.90-linux.sh --keep --noexec --nox11",
+        "patch -t ZeroBraneStudio-linux/install.sh install.sh.patch",
+        "patch -t ZeroBraneStudio-linux/app/zbstudio.sh zbstudio.sh.patch",
+        "patch -t ZeroBraneStudio-linux/app/zbstudio/res/zbstudio.appdata.xml zbstudio.appdata.xml.patch",
+        "patch -t ZeroBraneStudio-linux/app/zbstudio/res/zbstudio.desktop zbstudio.desktop.patch",
```

Comment:
you should just use `desktop-file-edit` here.

Reply:
Ok